netebpfext/net_ebpf_ext_hook_provider.c

```diff
@@ -284,6 +284,20 @@ net_ebpf_extension_hook_check_attach_parameter(
     NET_EBPF_EXT_RETURN_RESULT(result);
 }
 
+void
+_net_ebpf_extension_hook_client_cleanup(_Frees_ptr_opt_ net_ebpf_extension_hook_client_t* hook_client)
+{
+#pragma warning(push)
+#pragma warning(disable : 6001) // Using uninitialized memory '*hook_client'.
```

Comment:
Does this warning go way if you add the `_In_opt_` annotation to line 288?  I suspect that's the right fix.

Reply:
Yes that worked! Have pushed another commit.

But I am not sure why is it not giving error for (many) other similar instances. Example:
https://github.com/microsoft/ebpf-for-windows/blob/bebb9a14cd406d15bc312f12f5b1e628602d5647/netebpfext/net_ebpf_ext_prog_info_provider.c#L116